src/components/chip-set/chip.types.ts

```diff
@@ -25,6 +25,11 @@ export interface Chip<T = any> {
      */
     iconFillColor?: string;
 
+    /**
+     * `title` attribute of the icon
+     */
+    iconTitle?: string;
+
```

Comment:
It's probably time for us to add an `Icon` interface soon. A lot of the other components use the icon component and it creates a lot of duplications if all other components/interfaces has to add all these `iconXxx` properties. Instead we could have e.g. 
```ts
export interface Chip {
    icon?: string | Icon;
}

export interface Icon {
    name: string;
    color?: string;
    title?: string;
}
```
and reuse the `Icon` interface in the chip-set, menu, header component etc

It's nothing that has to be done in this PR, just making a general note about it :slightly_smiling_face:

Reply:
It is ok, can be fixed in this PR

Reply:
It's a bigger issue though and not related so it's probably better to separate it

Reply:
yes, let make an issue for icon interface so we don't forget; and fix it separately ;)